waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/FederatedHMSHandler.java

```diff
@@ -236,7 +241,8 @@ public void create_database(Database database)
   @Loggable(value = Loggable.DEBUG, skipResult = true, name = INVOCATION_LOG_NAME)
   public Database get_database(String name) throws NoSuchObjectException, MetaException, TException {
     LOG.info("Fetching database {}", name);
-    DatabaseMapping mapping = databaseMappingService.databaseMapping(name);
+    String internal_name = getDbInternalName(name);
```

Comment:
```suggestion
    String internalName = getDbInternalName(name);
```
etc. for the rest of the classes in this PR...